src/components/lab/Lattice.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LatticeRow from './LatticeRow';
+import { NUMBER_OF_ROWS_IN_LATTICE } from '../../config/constants';
+
+const Lattice = ({ stageDimensions }) => {
+  // create an array to map onto LatticeRow components
+  const rows = new Array(NUMBER_OF_ROWS_IN_LATTICE)
```

Comment:
I think you can use this `Array.from({length: 5}, (v, i) => i)`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

Reply:
Even better `[...Array(10).keys()]`